com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1816,6 +1823,11 @@ private List<TaskResult> runTasks(IProgress monitor) throws IOException {
                     monitor.worked(1);
                     continue;
                 }
+                if (isLastTask && !(task.getBuilder() instanceof GameProjectBuilder)) {
```

Comment:
If it's the last task, we check if it's `GameProjectBuilder` - it it's not, the build pipeline is broken.